user_guide/intro_nav_pinned_pages.rst

```diff
@@ -0,0 +1,61 @@
+.. _user-guide-ui-components-pinned:
+
+Pinbar
+======
+
+Pinning pages is a convenient tool that allows you to save a page and easily return to it. When you open a pinned page, 
+it will look exactly as it did when you have pinned it.
+
+Pins are particularly  useful to:
+
+* Save unfinished forms if you need to switch to another page or simply take a break in your work. All the information 
+  that was in the form at the moment you’ve pinned it will be preserved even if you log out and then log in from another 
+  device.
+
+* Save frequently used filtered grids and search results.
+
+* Pin the most important records to keep an eye on them. 
+
+Pin a page
+----------
+
+You can pin a page to the header and come back to it at any time in one click. Some pages, such as 
+:ref:`Dashboards <user-guide-dashboards>` or :ref:`Calendars <user-guide-calendars>` cannot be pinned.
+
+To pin a page, click the pin icon in the top right corner of the page. 
+
+|
+
+.. image:: ./img/navigation/panel/pin.png  
+
+|
+
+When you’ve pinned a page the pin icon will turn yellow and the link to a page will appear in the pin bar on the top. 
```

Comment:
Edit: When you’ve pinned a page, the pin icon will turn yellow and a link to the page will appear in the pin bar at the top.